app/assets/stylesheets/_color_variables.scss

```diff
@@ -0,0 +1,3 @@
+$color-title-main: white;
+$color-body: #E3DEC1;
+$color-buttons-main: #eee;
```

Comment:
You might want to choose one style of describing colors to use throughout a project.